interfaces/cython/cantera/ck2yaml.py

```diff
@@ -670,9 +670,9 @@ def __init__(self, *, A, B, C, D=None, E=None):
 
     def reduce(self, output):
         sri = FlowMap([('A', self.A), ('B', self.B), ('C', self.C)])
-        if self.D:
+        if self.D != None:
```

Comment:
```suggestion
        if self.D is not None:
```

Reply:
Agreed. `None` should not be compared with `!=` ([PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)?). Otherwise I'm happy that you were able to confirm my suspicion about the bug report.